### PR TITLE
Support for PostgreSQL with version bump

### PR DESCRIPTION
**Pull request by cheziyi**

### CODE REVIEW

File: PrismaDB-QueryAST/PrismaDB-QueryAST.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>0.2.8.0</Version>
+    <Version>0.1.0.0</Version>
```
Comment (bazzilic) [inline PrismaDB-QueryAST/PrismaDB-QueryAST.csproj]: This looks more like a version downgrade, from 0.2 to 0.1.

**Comment by cheziyi** [comment]
This is so we set a generic version number for csproj, and use only appveyor to bump to make it cleaner

File: PrismaDB-QueryAST/PrismaDB-QueryAST.csproj (hunk 1 above)
Comment (bazzilic) [inline PrismaDB-QueryAST/PrismaDB-QueryAST.csproj]:
```suggestion
    <Version>1.0.0.0</Version>
```

Comment (bazzilic) [reply PrismaDB-QueryAST/PrismaDB-QueryAST.csproj]: This looks more generic, just to keep it neat

Comment (cheziyi) [reply PrismaDB-QueryAST/PrismaDB-QueryAST.csproj]: This is actually functional as well, for local testing it will supercede nuget packages

Comment (bazzilic) [reply PrismaDB-QueryAST/PrismaDB-QueryAST.csproj]: Ok, other than that, looks good

**Pull request merged**